src/geometry/3d_intersection_of_planes.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+typedef pair<Point3d,bool> P3db;
```

Comment:
```suggestion
using P3db = pair<Point3d, bool>;
```
型の別名には`using`を使うのがモダンのようです（同上）。 [参考](https://github.com/cpprefjp/site/issues/394)